### PR TITLE
Fix #157

### LINKED ISSUE #157: question: maintainer_deprecated test is passing also with MAINTAINER inside Dockerfile
I've looked at test:
https://github.com/user-cont/colin/blob/master/colin/checks/dockerfile.py
and I've modified Dockerfile to contain ``MAINTAINER`` I've tries to use ``MAINTAINER "asdasd"`` and also ``MAINTAINER=asdgad``

```
FROM registry.fedoraproject.org/fedora:27

ENV NAME=colin-labels \
    VERSION=0.0.4

MAINTAINER "asdadf@asd"

LABEL name="${NAME}" \
    summary="Colin image used for testing Dockerfile labels" \
    maintainer="Petr Hracek <phracek@redhat.com>" \
    version="${VERSION}" \
    com.redhat.component="colin-labels" \
    description="The image contains labels which are used for testing colin functionality." \
    io.k8s.description="The image contains labels which are used for testing colin functionality." \
    run="docker run <application image>"

COPY ./files/usage /files/usage

CMD ["/files/usage"]
```
and ``colin check tests/data/Dockerfile`` **PASSed**
```
PASS:Label 'maintainer' has to be specified.
PASS:In FROM, tag has to be specified and not 'latest'.
PASS:Dockerfile instruction `MAINTAINER` is deprecated.

PASS:3 
```

is that okay, or this test is somehow bad. I've found that when working on FMF metadata and found that it passed in my PR, although it should not. But when I've used master branch it also **PASSed**

### PR DESCRIPTION
**Pull request by TomasTomecek**
Fixes #157

Commits:

* fix instruction checks when *_count is 0
* tests: more verbose logs
* enable all tests by default

### CODE REVIEW

File: tests/integration/test_labels.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def empty_ruleset():
 
 
 def get_results_from_colin_labels_image():
-    return colin.run("colin-labels", ruleset_name="fedora")
+    return colin.run("colin-labels", ruleset_name="fedora", logging_level=10)
```
Comment (jscotka) [inline tests/integration/test_labels.py]: maybe would be better to use constants inside logging module ``logging_level=logging.INFO``

Comment (TomasTomecek) [reply tests/integration/test_labels.py]: this way I don't have to import logging :P

Comment (TomasTomecek) [reply tests/integration/test_labels.py]: anyone, fixed

Comment (jscotka) [reply tests/integration/test_labels.py]: little bit unclear,  but makes sense :-)

**Pull request merged**